spec/features/admin/settings_spec.rb

```diff
@@ -89,4 +89,48 @@
 
   end
 
-end
+  describe "Skip verification" do
+
+    scenario "deactivate verification", :js do
+      Setting["feature.user.skip_verification"] = 'true'
```

Comment:
Again, setting and resetting a `Setting` value in a before & after blocks are the right way as you can see in allmost all  the codebase specs using this kind of behaviour. Please read similar specs to understand how to tackle a particular spec/code, you'll run into this problems again and again if not.